langchain/src/chat_models/anthropic.ts

```diff
@@ -159,7 +164,14 @@ export class ChatAnthropic extends BaseChatModel implements AnthropicInput {
   // Used for streaming requests
   private streamingClient: Anthropic;
 
-  constructor(fields?: Partial<AnthropicInput> & BaseChatModelParams) {
+  private clientOptions: ClientOptions;
+
+  constructor(
+    fields?: Partial<AnthropicInput> &
+      BaseChatModelParams & {
+        clientOptions?: ClientOptions;
```

Comment:
Could we instead add `clientOptions?` to `AnthropicInput` directly?

Reply:
Done & added test